app/pods/components/input/checkbox/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="input checkbox {{@value}} {{if @saving 'saving'}}" {{on 'click' this.toggle}}>
+  <div>
+    {{#if @saving}}
+      <Svg::Ellipsis @fill="#fff" />
+    {{else if @value}}
+      <Svg::Check @fill="#fff" />
```

Comment:
Good approach. This is what I started out with, but then reverted from it.
1. The template doesn't care about styling, CSS files do, separation of concerns.
2. By fixating the colour in the HTML, we aren't able to leverage CSS `:hover` and `@media` to set different colours.

Reply:
I understand, makes sense!